<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogStatus</class>
 <widget class="QDialog" name="DialogStatus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>状态</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="cbAutoGet">
      <property name="text">
       <string>自动刷新</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>刷新周期(ms):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leAutoGetInterval">
        <property name="text">
         <string>1100</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>电池电压(V):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leBateryVoltage">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>160</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>电机油门:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="teThrottle">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btGet">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btGet</tabstop>
  <tabstop>cbAutoGet</tabstop>
  <tabstop>leAutoGetInterval</tabstop>
  <tabstop>leBateryVoltage</tabstop>
  <tabstop>teThrottle</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
